src/status_im/contexts/wallet/send/routes/view.cljs

```diff
@@ -201,12 +201,13 @@
       :on-press-to-network   on-press-to-network}]))
 
 (defn- view-internal
-  [{:keys [from-values-by-chain to-values-by-chain routes token theme fetch-routes
+  [{:keys [from-values-by-chain to-values-by-chain routes token theme fetch-routes input-value
            affordable-networks disabled-from-networks on-press-from-network on-press-to-network]}]
   (let [token-symbol              (:symbol token)
         loading-suggested-routes? (rf/sub [:wallet/wallet-send-loading-suggested-routes?])
         network-links             (if loading-suggested-routes? affordable-networks routes)]
-    (if (or (and (not-empty affordable-networks) loading-suggested-routes?) (not-empty routes))
+    (if (and (not-empty input-value)
```

Comment:
it's kind of tricky to follow this `if` statement 🤔 
It might be easier to see if you separate this `let` block below into a separate component? 
maybe there's some other adjustment to make it easier to follow

Reply:
I agree, this component is complicated but I think we can simplify it by modifying its approach by getting rid of FlatList and pre-calculating things on the events side, while making to actually support all the different possible scenarios. I've created an issue (#19696) which I will prioritize because it is functionality we need to have a more solid implementation for. Refactor will come with its PR to this component and probably will improve the code quality 👍

Reply:
Thanks!